release/ray_release/command_runner/_anyscale_job_wrapper.py

```diff
@@ -65,7 +65,7 @@ def run_storage_cp(source: str, target: str):
         return False
 
     if not Path(source).exists():
-        logger.error(f"Couldn't upload to cloud storage: '{source}' does not exist.")
+        logger.warning(f"Couldn't upload to cloud storage: '{source}' does not exist.")
```

Comment:
could you give an example link?

also how are warning and errors handled differently today?